tests/PactContinuationSpec.hs

```diff
@@ -249,17 +249,18 @@ threeStepPactCode moduleName =
 threeStepNestedPactCode :: T.Text -> T.Text
 threeStepNestedPactCode moduleName =
   [text|
-    (define-keyset 'k (read-keyset "admin-keyset"))
     (interface iface
 
      (defpact ndp:string ())
     )
-    (module $moduleName 'k
+    (module $moduleName g
```

Comment:
How does this change relate to native versioning?

Reply:
Great question.

The flag which runs this introduces namespaced keysets. We can't just `define-keyset`. 

We're not testing for that stuff at all.